src/Redistribution.sol

```diff
@@ -372,9 +377,10 @@ contract Redistribution is AccessControl, Pausable {
         bytes32 obfuscatedHash = wrapCommit(_overlay, _depth, _hash, _revealNonce);
         uint256 id = findCommit(_overlay, obfuscatedHash);
         Commit memory revealedCommit = currentCommits[id];
+        uint8 depthResponsibility = _depth - revealedCommit.height;
```

Comment:
there is no need to subtract here. we should use _depth directly. 
bee will submit it's current storage radius, which will have height already subtracted from it.